filebeat/module/haproxy/http/test/haproxy.log-expected.json

```diff
@@ -1,47 +1,44 @@
 [
     {
-        "haproxy.server_name": "docs",
-        "haproxy.time_client_req": 0,
-        "haproxy.geoip.continent_name": "North America",
-        "haproxy.geoip.city_name": "Mukilteo",
-        "haproxy.geoip.country_iso_code": "US",
-        "haproxy.geoip.region_name": "Washington",
-        "haproxy.geoip.location.lon": -122.3042,
-        "haproxy.geoip.location.lat": 47.913,
-        "haproxy.termination_state": "----",
-        "haproxy.time_queue": 0,
-        "haproxy.pid": 32450,
-        "haproxy.client_port": 38862,
-        "haproxy.backend_queue": 0,
-        "haproxy.process_name": "haproxy",
-        "haproxy.backend_name": "docs_microservice",
-        "haproxy.http.request.raw_request_line": "GET /component---src-pages-index-js-4b15624544f97cf0bb8f.js HTTP/1.1",
-        "haproxy.http.request.captured_cookie": "-",
+        "@timestamp": "2018-07-30T09:03:52.726Z", 
```

Comment:
Looks like we should make sure the output fields are always sorted the same way.

Reply:
When `GENERATE=1`is used to generate these files, they are alphabetically sorted.
```
                json.dump(objects, f, indent=4, sort_keys=True)
```
This file came from community, not sure how it was generated then, but it should be fine in that sense from now on.